src/components/Toolbar/FormattingToolbar.js

```diff
@@ -77,7 +77,9 @@ class FormattingToolbar extends React.Component<Props> {
 
     const data = { href: "" };
     this.props.editor.wrapInline({ type: "link", data });
-    this.props.onCreateLink(ev);
+    if (data.href.length) {
```

Comment:
You're checking a value that is set to an empty string above on L78  so this callback will never be triggered I'm afraid.

Reply:
It might be that we want to check the length of the selection before ever calling `handleCreateLink`? 🤔

Reply:
Right haha, i'm not used to react but i managed to get the selection.  I've already pushed my new commit, what do you think? Makes the calling of the trim function sense?